### PR TITLE
Fix cleanup kmesh-cni failed bug

### LINKED ISSUE #365: kmesh-cni and kmesh-cni-kubeconfig remove failed when uninstall kmesh
**What happened**:
when uninstall kmesh, it can not remove kmesh cni and kmesh-cni-kubeconfig correctly.

It report an error `level=error msg="err:remove /opt/cni/bin/kmesh-cni-kubeconfig: no such file or directory, remove CNI config failed, please remove manual" subsys="cni installer"`

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kmesh version:
- Others:

### PR DESCRIPTION
**Pull request by bfforever**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Fix cleanup kmesh-cni failed bug.

**Which issue(s) this PR fixes**:
Fixes  #365 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/cni/install.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func (i *Installer) Stop() {
 	if i.Mode == constants.AdsMode || i.Mode == constants.WorkloadMode {
 		log.Info("start remove CNI config\n")
 		if err := i.removeCniConfig(); err != nil {
-			log.Error("remove CNI config failed, please remove manual")
+			log.Errorf("err:%v, remove CNI config failed, please remove manual", err)
```
Comment (LiZhenCheng9527) [inline pkg/cni/install.go:83]: In line 81 of the log, a newline character was added. Does it need to be added here? What is the standard for adding or not adding newline characters in Kmesh logs?

Comment (hzxuzhonghu) [inline pkg/cni/install.go:83]:
```suggestion
			log.Errorf("remove CNI config failed: %v, please remove manually", err)
```

Comment (hzxuzhonghu) [reply pkg/cni/install.go:83]: we donot need to add \n by hand, becasue  log.Errorf/Info/Debug has also added \n implicitly

**Comment by hzxuzhonghu** [comment]
@bfforever  Yare qualified to apply member https://github.com/kmesh-net/kmesh/issues/new?assignees=&labels=&projects=&template=membership-request.md&title=REQUEST%3A+New+membership+for+%3Cyour+name%3E

**Pull request merged**